### DISCUSSION TITLE
使用Vscode的xmake插件遇到了一点问题，能够run，但是单步调试失败

### DISCUSSION THREAD

**Discussion by axxdzkl** [xmake-io/xmake-vscode#274] [Q&A]
![image](https://github.com/user-attachments/assets/cba848f0-8809-40fa-ae7d-efc0dcd4eb73)
在hello中，能运行，但是调试失败。Vscode自己的调试器能够单步调试
![image](https://github.com/user-attachments/assets/e536d6b4-7bfe-445a-9a44-9b3f5ef9da30)

**Comment by waruqi** [comment]
vscode 没有自己的调试器，都是用的 cpptools 或者 codelldb 的。。xmake 的插件也是调用这两的调试器。

具体问题你自己调下插件源码
